projects/src/simbicycle.hh

```diff
@@ -112,6 +113,7 @@ void Bicycle<Model, Observer, Haptic>::update_dynamics(real_t roll_torque_input,
         m_observer.set_state(x);
     }
 
+    m_input = input;
```

Comment:
Is there a good reason to create a new `input_t input` and copy it into `m_input` here?